sql/core/src/test/java/test/org/apache/spark/sql/JavaDataFrameSuite.java

```diff
@@ -328,7 +328,7 @@ public void testTextLoad() {
 
   @Test
   public void testCountMinSketch() {
-    Dataset df = context.range(1000);
+    Dataset<Object> df = context.range(1000);
```

Comment:
To resolve the warning it needs a generic type, but `Dataset<?>` may be sufficient here.